modules/ROOT/pages/import-api-task.adoc

```diff
@@ -1,13 +1,19 @@
-= Creating an API instance by importing on API Manager
+= Creating an API Instance by Importing to API Manager
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-You can develop an API for and export it from API Manager for future use or for sharing. The exported API can be imported using this procedure. To perform this task, you need to have a ZIP file on your file system containing an API object exported from API Manager.
+You might have developed an API and exported it from API Manager to share it across your organization or to use it at a later time. You can then import this API into API Manager when required. 
 
-. After logging into Anypoint Platform, click API Manager.
-. In API Administration, select *Manager API > Import API from the Desktop*.
-. Browse to and select the ZIP file that contains the exported API instance to import.
+== Task Prerequisite
+
+Before you import an API to API Manager, you must have already exported the API instance ZIP file to you file system on your local machine or a server.
+
+To import the API instance to API Manager:
+
+. Go to *API Manager > API Administration*.
+. From the main page, select *Manage API > Import*.
+. In the file browser, select the API instance ZIP file to import.
```

Comment:
How did the user get to a browser?
This term might be misleading.
Can you use only...:
```suggestion
. Select the ZIP file of the API that you want to import.
```

Reply:
Added it to the previous step explained.

Reply:
Done! Thanks!